app/assets/styles/account/details.scss

```diff
@@ -5,19 +5,14 @@ bl-account-details {
 
         .link {
```

Comment:
cloud maybe have a bit less padding for those tiles(And also maybe rename from `.link`